docs/extra-apps.md

```diff
@@ -0,0 +1,8 @@
+# Extra apps
+
+## omxplayer
+
+omxplayer depends on libav which has a commercial license. So in order to be
+able to compile omxplayer you will need to whiteflag the commercial
+licenseadding to you local.conf:
+`LICENSE_FLAGS_WHITELIST = "commercial"`
```

Comment:
This looks strange to me. The output is something like this:

licenseadding to you local.conf:
`LICENSE_FLAGS_WHITELIST = "commercial"`

I think this looks better

licenseadding to you local.conf:

    LICENSE_FLAGS_WHITELIST = "commercial"

Above is a new line and indented 4 spaces. There are a lot of places across the documents where my alternative would look better and improve read-ability. IMO.